examples/ts-lb-simple/index.ts

```diff
@@ -2,14 +2,51 @@ import * as pulumi from "@pulumi/pulumi";
 import * as aws from "@pulumi/aws";
 import * as awsx from "@pulumi/awsx";
 
+// Create VPC for the LB and ECS service
```

Comment:
The changes to the examples were required because they heavily used invokes  (e.g. to get the default VPC) which are not compatible with upgrade tests because they do call out to the cloud. This caused the tests to fail in CI because it was running in a different AWS account.

This is what caused https://github.com/pulumi/pulumi-awsx/issues/1114